torch/csrc/autograd/python_variable.cpp

```diff
@@ -214,7 +214,7 @@ static PyObject * THPVariable_get_data(THPVariable *self)
   /// make such changes explicitly illegal, in order to prevent users from changing
   /// metadata of the `.data` tensor and expecting the original tensor to also
   /// be updated.
-  auto var = make_variable(self->cdata.data(), /*requires_grad=*/false, /*allow_tensor_metadata_change=*/false);
+  auto var = make_variable(self->cdata, /*requires_grad=*/false, /*allow_tensor_metadata_change=*/false);
```

Comment:
It's okay to remove the `.data()` call here, because `make_variable(...)` always ignores `self->cdata`'s original AutogradMeta and creates a new AutogradMeta.